config/joystick.yaml

```
@@ -0,0 +1,27 @@
+    axis_angular:
+      yaw: 3
+    scale_angular:
+      yaw: 0.5
+    scale_angular_turbo:
+      yaw: 1.0
+
+
+    # enable_button: 4
+    enable_turbo_button: 5
+
+    require_enable_button: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)